Userland/Libraries/LibJIT/CMakeLists.txt

```diff
@@ -4,6 +4,9 @@ set(SOURCES
 )
 if(NOT APPLE AND NOT WIN32 AND NOT EMSCRIPTEN)
     list(APPEND SOURCES ELFBuild.cpp)
+    list(APPEND SOURCES GDB-elf.cpp)
+else()
+    list(APPEND SOURCES GDB-unsupported.cpp)
```

Comment:
We don't use this file naming style anywhere else in the system, I don't think. GDBELF or GDB_ellf (GDBImageELF?) would match the rest of the project better.